apps/src/templates/HourOfCodeGuideEmailDialog.jsx

```diff
@@ -97,40 +90,44 @@ function HourOfCodeGuideEmailDialog({isSignedIn, unitId}) {
           <div className={style.middle}>
             <Heading3>{i18n.learnHowToHost()}</Heading3>
             {bodyText}
-            <label className={style.typographyLabel}>
-              {i18n.yourNameCaps() + '*'}
-              <input
-                required
-                type="text"
-                id="uitest-hoc-guide-name"
-                className={style.classNameTextField}
-                value={name}
-                onChange={e => setName(e.target.value)}
-              />
-            </label>
-            <label className={style.typographyLabel}>
-              {i18n.yourEmailCaps() + '*'}
-              <input
-                required
-                type="text"
-                id="uitest-hoc-guide-email"
-                className={style.classNameTextField}
-                value={email}
-                onChange={e => setEmail(e.target.value)}
-              />
-            </label>
-            <label className={style.label}>
-              <input
-                checked={isMarketingChecked}
-                className={style.box}
-                type="checkbox"
-                id="uitest-receive-updates-checkbox"
-                onChange={() => {
-                  setIsMarketingChecked(!isMarketingChecked);
-                }}
-              />
-              {i18n.receiveFutureUpdates()}
-            </label>
+            {!isSignedIn && (
```

Comment:
I'd forgotten to add this when I added the fields: we don't need to gather this info if we already have the user! It's just the conditional and the div; there are no other changes